front/pages/w/[wId]/vaults/[vaultId]/categories/[category]/index.tsx

```diff
@@ -54,12 +55,12 @@ export const getServerSideProps = withDefaultUserAuthRequirements<
     auth,
     context.query.vaultId as string
   );
-  if (!vault || !systemVault) {
+  if (!vault || !systemVault || (vault.canRead(auth))) {
```

Comment:
Typo here, we throw 404 if can read 😅 

Why do we load both vault & systemVault? To get a link to system vault?